carma-messenger-core/truck_inspection_plugin/config/parameters.yaml

```diff
@@ -1,2 +1,2 @@
 # Number of Key-Value entries in safety log
-num_of_entries: 14
+num_of_entries: 13
```

Comment:
What changed from last update of 14 parameters to 13?

Reply:
According to this https://github.com/usdot-fhwa-stol/carma-platform/blob/3e6684787318a918ab2faf1fb71812014a990a53/truck_inspection_client/src/truck_inspection_client.cpp#L88. There are only 13 keys in the strategy params. So it is an existing issue.